examples/electron/package.json

```diff
@@ -14,17 +14,25 @@
     "theia": "file:../../../Theia"
   },
   "scripts": {
-    "install:theia": "cd ../ && npm link && cd examples && npm link theia",
-    "clean": "./node_modules/.bin/rimraf lib",
-    "lint": "./node_modules/.bin/tslint -c ../../config/tslint/tslint.json --project ./tsconfig.json",
-    "build": "npm run build:app && npm run build:web",
-    "build:app": "./node_modules/.bin/tsc && npm run lint",
-    "build:electron": "./node_modules/.bin/electron-rebuild && npm run build:app && ./node_modules/.bin/webpack --target electron && cp src/client/index.html lib/client",
-    "start:app": "./node_modules/.bin/node ./lib/app.js",
-    "start:electron": "./node_modules/.bin/electron ./lib/client/electron-main.js",
-    "cold:start:electron": "cd ../../ && npm run clean && npm run build && cd examples/electron && npm run clean && npm run build:app && npm run build:electron && npm run start:electron",
-    "watch:app": "./node_modules/.bin/tsc --watch",
-    "test:ui": "./node_modules/.bin/wdio wdio.conf.js"
+    "clean": "rimraf lib",
+
+    "postinstall": "electron-rebuild",
+    "prepare": "npm run build",
+
+    "build": "npm run build:app && npm run build:electron",
+    "build:app": "tsc && npm run lint",
+    "build:electron": "npm run build:app && webpack --target electron && cp src/client/index.html lib/client",
+    "lint": "tslint -c ../../config/tslint/tslint.json --project ./tsconfig.json",
+
+    "start": "npm run start:electron",
+    "start:app": "node ./lib/app.js",
+    "start:electron": "electron ./lib/client/electron-main.js",
+
+    "cold:start:electron": "cd ../../ && npm run clean && npm run build && cd examples/electron && npm run clean && npm run build && npm start",
```

Comment:
In the browser version though, we have:

 - `start:app`: start the backend
 - `start:browser`: start the frontend
 - `start`: start both

Here I'd keep start:browser, because you might want to start the frontend only.



```